src/components/avatar-group/AvatarGroup.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <button @click="$emit('click')" class="avatar-group">
+    <j-tooltip title="See all members">
+      <div class="avatar-group__avatars">
+        <j-avatar
+          v-for="(user, index) in firstUsers"
+          :key="index"
+          :src="require('@/assets/images/junto_app_icon.png')"
+          size="sm"
+        ></j-avatar>
+        <span v-if="users.length >= 4" class="avatar-group__see-all">
+          + {{ users.length - 3 }}
+        </span>
+      </div>
+    </j-tooltip>
+  </button>
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  emits: ["click"],
+  props: ["users"],
+  computed: {
+    firstUsers(): any {
+      return [{ ...this.users[0] }, { ...this.users[1] }, { ...this.users[2] }];
```

Comment:
could use split here `this.users.split(2)`

Reply:
Good point, thanks